### DISCUSSION TITLE
Include "babel-plugin-react-remove-properties" functionality

### DISCUSSION THREAD

**Discussion by leandroluk** [swc-project/swc#6057] [General]
### Describe the feature

When creating a web project, it is quite common to use the data-testid attribute to identify elements in components. In a production environment these attributes only serve to pollute and increase the size of the final file and to solve the problem there is the plugin [babel-plugin-react-remove-properties](https://www.npmjs.com/package/babel-plugin-react-remove-properties)

The way to use this plugin is very simple see:

```javascript
const isPrd = process.env.NODE_ENV === 'production'

const devPlugins = []
const prdPlugins = [["react-remove-properties", { "properties": ["data-testid"] }]]

module.exports = {
  presets: ["next/babel"],
  plugins: isPrd ? prdPlugins : devPlugins
}
```

### Babel plugin or link to the feature description

https://www.npmjs.com/package/babel-plugin-react-remove-properties

### Additional context

This type of resource is important to reduce the size of the pages so it optimizes SEO

**Comment by kwonoj** [comment]
Babel itself does not provides this kind of features but 3rd party plugin offers, I believe this should be written in SWC plugin as well.
